### ISSUE TITLE
Add Support for Swift Package Manager

### ISSUE THREAD

**Issue by MohanabalanD** [Skyscanner/SkyFloatingLabelTextField#333]
# Add Support for Swift Package Manager

Since Most of the projects are Migrating to Swift Package Manager, we want this wonderful Library to add Support for Swift Package Manager

**Comment by vishnupxminds** [comment]
Yes It would be great if you guys can add support for Swift Package Manager
